### ISSUE TITLE
Border around reputation notification badge

### ISSUE THREAD

**Issue by AfroThundr3007730** [StylishThemes/StackOverflow-Dark#170]
Should it stay white as it is now, or be darker to blend with the current theme. The border definitely draws attention to it, but it may be overkill.

White mode:
![a](https://user-images.githubusercontent.com/7613032/61315628-b8715180-a7ee-11e9-92a0-d2559e7dd579.png)

Dark mode:
![b](https://user-images.githubusercontent.com/7613032/61315645-be673280-a7ee-11e9-83b4-50e803743cac.png)

* **Browser**: Firefox Nightly 70.0a1 2019-07-16
* **Operating System**: Debian Testing 11 x86_64
* **Link to page with the issue**: N/A
* **Screenshot**: Attached above

**Comment by the-j0k3r** [comment]
I think if the border is dark it will look like intended.

**Comment by AfroThundr3007730** [comment]
Agreed. Not sure where you want it added (around L537?) but here is the rule needed.
```css
.top-bar .indicator-badge {
  border-color: black;
}
```

**Comment by Mottie** [comment]
It's easier to make the border transparent so it wouldn't need to be adjusted in the future.

**Comment by the-j0k3r** [comment]
> Agreed. Not sure where you want it added (around L537?) but here is the rule needed.

Just add it to a transparent border-color rule (as per @Mottie suggestion) that isnt !important (if it works and !important if it doesnt)

**Comment by AfroThundr3007730** [comment]
About to send a PR. One thing though, setting it transparent looks like so:
![transparent](https://user-images.githubusercontent.com/7613032/61390680-c4bae480-a8aa-11e9-9028-b02df83b23a8.png)

Whereas setting it black looks like this:
![black](https://user-images.githubusercontent.com/7613032/61390701-cab0c580-a8aa-11e9-8ac9-ce3a7b362efd.png)

The second version looks more in line with the intended style from the original, where the badge was detached from the icon it's hovering over. It still stands out either way, so whichever you guys want.
